source/_components/mychevy.markdown

```diff
@@ -40,6 +40,10 @@ password:
   description: The password for your given my.chevrolet account.
   required: true
   type: string
+country:
+  description: Which country's servers to contact. Defaults to 'us', also supports 'ca'.
```

Comment:
Put the default in a `variable`
So: "default: us"

Reply:
Thanks!